Jenkinsfile

```diff
@@ -123,6 +126,41 @@ pipeline{
 								sh "scripts/dev_cli.sh tests --integration --libc musl"
 							}
 						}
+						stage ('Install azure-cli') {
+							steps {
+								installAzureCli("bionic", "arm64")
+							}
+						}
+						stage ('Download Windows image') {
+							steps {
+								sh '''#!/bin/bash -x
+									IMG_BASENAME=windows-11-iot-enterprise-aarch64.raw
+									IMG_PATH=$HOME/workloads/$IMG_BASENAME
+									IMG_GZ_PATH=$HOME/workloads/$IMG_BASENAME.gz
+									IMG_GZ_BLOB_NAME=windows-11-iot-enterprise-aarch64-9-min.raw.gz
+									IMG_SHA1_SUM_FL="$IMG_PATH.sha1"
+									if test -f "$IMG_SHA1_SUM_FL" && sha1sum -c "$IMG_SHA1_SUM_FL"; then
+										exit
+									fi
+									mkdir -p "$HOME/workloads"
+									az storage blob download \
+										--container-name private-images \
+										--file "$IMG_GZ_PATH" \
+										--name "$IMG_GZ_BLOB_NAME" \
+										--connection-string "$AZURE_CONNECTION_STRING"
+									gzip -d $IMG_GZ_PATH
+									sha1sum "$IMG_PATH" > "$IMG_SHA1_SUM_FL"
```

Comment:
I would have thought it makes most sense to check this file into the source tree? All this seems to do is check that the file hasn't changed on disk since it was downloaded / skip the download?

Reply:
Indeed. I checked it in now. First idea was to just calculate once at after the initial download, but that actually might be broken by chance. 

Thanks